polkadot/runtime/test-runtime/src/lib.rs

```diff
@@ -94,7 +94,7 @@ use sp_staking::SessionIndex;
 #[cfg(any(feature = "std", test))]
 use sp_version::NativeVersion;
 use sp_version::RuntimeVersion;
-use xcm::v4::{Assets, InteriorLocation, Location, SendError, SendResult, SendXcm, XcmHash};
+use xcm::v5::{Assets, InteriorLocation, Location, SendError, SendResult, SendXcm, XcmHash};
```

Comment:
@franciscoaguirre yes, yes, my favorite, I think we should use `xcm::latest` here :D

@seadanda I also think we should refactor [`polkadot/runtime/parachains/src/coretime/mod.rs`](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/coretime/mod.rs#L35) to use `xcm::latest` instead of the specific `xcmV4` version. If I understand correctly, this pallet only [sends](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/coretime/mod.rs#L402) some XCM at some point and does not store XCM data/structs, so it should be fine and preferable to use `xcm::latest`.

Keeping this pallet on `xcmV4` while migrating the rest to `xcmV5` could lead to potential issues, as we would need to manage multiple imports in the fellows' runtimes, e.g., `xcm::v4::SendXcm` and `xcm::v5::SendXcm`.

For now, it’s fine on the master since `xcmV4` is the latest version. However, I recommend refactoring the coretime pallet directly on the master branch before merging `xcmV5`.